### PR TITLE
Added GE Z-Wave Direct-Wire Outdoor/Indoor Smart Switch

### PR DESCRIPTION
**Pull request by sofakng**
Added GE Z-Wave Direct-Wire Outdoor/Indoor Smart Switch.

See: https://products.z-wavealliance.org/products/2939

### CODE REVIEW

File: config/ge/14285.xml [hunk 1]
```diff
@@ -0,0 +1,58 @@
+<Product Revision="1" xmlns="https://github.com/OpenZWave/open-zwave">
+<MetaData>
+    <MetaDataItem name="OzwInfoPage">http://www.openzwave.com/device-database/0063:3032:4F44</MetaDataItem>
+    <MetaDataItem name="ProductPic">images/ge/14285.png</MetaDataItem>
```
Comment (Fishwaldo) [inline config/ge/14285.xml:4]: This image is missing from the commit

**Comment by sofakng** [comment]
Added missing image.  Sorry!

File: config/ge/14285.xml (hunk 1 above)
Comment (sofakng) [reply config/ge/14285.xml:4]: Fixed.

**Comment by Fishwaldo** [comment]
Thanks!

**Pull request merged**